src/OOVPADatabase/D3D8LTCG/5849.inl

```diff
@@ -210,6 +235,61 @@ OOVPA_SIG_MATCH(
     //
 );
 
+//test case: Otogi 2/MechAssault 2
```

Comment:
nit: Add space and move below the title symbol comment, not above. testcase is one word for internal documentation.